packages/host/app/resources/auto-attached-card.ts

```diff
@@ -82,13 +110,19 @@ export class AutoAttachment extends Resource<Args> {
 export function getAutoAttachment(
   parent: object,
   args: {
+    submode: () => Submode;
+    autoAttachedFileUrl: () => string | undefined;
+    playgroundPanelCardId: () => string | undefined;
```

Comment:
why is this treated specially? can't we just collapse values here into the `attachedCardIds` thunk?